clusters/app.ci/kni-edge/admin_rbac.yaml

```diff
@@ -35,6 +35,22 @@ users:
 - mcornea
 - yprokule
 ---
+# this grants the right to read the ServiceAccount's credentials
+kind: Role
```

Comment:
A role without binding doesn't do anything, can you also update your binding?

Reply:
thanks @alvaroaleman - added new binding ptal if that's OK

Reply:
No, update the existing bindings roleRef, not adding a new one

Reply:
Ack, updated existing